### DISCUSSION TITLE
Remove useless information from console when running `npx prisma generate` and equivalents

### DISCUSSION THREAD

**Discussion by incompletude** [prisma/prisma#11395] [Q&A]
When I run `npx prisma genereate` the console shows me:

```
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma

✔ Generated Prisma Client (3.8.1 | library) to ./node_modules/@prisma/client in 657ms

✔ Generated DBML Schema to ./prisma/dbml in 51ms

You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client

import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()
```

Is there a way to remove the useless part?

```
You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client

import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()
```

This is useless information 99.9% of the time.

**Comment by l2D** [comment]
Hello, @incompletude 

In my thought, this info helps me to know which version and schema I used when I've mono repositories. I think it is quite helpful.

**Reply by incompletude** [reply]
How so?

**Reply by pantharshit00** [reply]
Hello @incompletude, our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

I will recommend you to open a feature request where this can be properly tracked and someone from cli team can comment on this there: https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=

**Reply by janpio** [reply]
Already tracked: https://github.com/prisma/prisma/issues/3390

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
